### PR TITLE
Main page text update

### PR DESCRIPTION
**Pull request by priyanka-surana**

### CODE REVIEW

**Comment by muffato** [comment]
I rewrote / rearranged text on the front page to make the content more about what _we_ value.

I rebased my branch onto `goat` to have the latest version of the tools page, which I quickly expanded by adding a few more tools, just to flesh it out.

**Comment by priyanka-surana** [comment]
I can resolve the GoaT conflict and do the deployment. But for the new software added, the formatting is not consistent and links in the header will not show up well. The web page rendering is different from GitHub.

**Comment by priyanka-surana** [comment]
@muffato If you are good with the changes I made, I am going to merge this today.

File: public_html/index.php [hunk 1]
```diff
@@ -436,7 +428,7 @@
       <div class="col-sm-6">
         <h2 id="get-started" class="text-white">Get started in minutes</h2>
         <p class="text-white-50">Nextflow lets you run sanger-tol pipelines on virtually any computing environment.</p>
-        <p class="text-white-50">All sanger-tol genomics pipelines come with built-in support for
+        <p class="text-white-50">All sanger-tol genomic pipelines are modifiable with the
```
Comment (muffato) [inline public_html/index.php:431]: We're using nf-core for pipelines in general, not just the genomics ones (even though everything we do is genomics)
```suggestion
        <p class="text-white-50">All sanger-tol pipelines are modifiable with the
```

**Pull request merged**